pkg/netflow/flowaggregator/aggregator.go

```diff
@@ -98,7 +108,14 @@ func (agg *FlowAggregator) sendFlows(flows []*common.Flow) {
 		}
 
 		log.Tracef("flushed flow: %s", string(payloadBytes))
-		agg.sender.EventPlatformEvent(payloadBytes, epforwarder.EventTypeNetworkDevicesNetFlow)
+
+		m := &message.Message{Content: payloadBytes}
+		err = agg.epForwarder.SendEventPlatformEventBlocking(m, epforwarder.EventTypeNetworkDevicesNetFlow)
+		if err != nil {
+			// at the moment, SendEventPlatformEventBlocking can only fail if the event type is invalid
+			log.Errorf("Error sending to event platform forwarder: %s", err)
```

Comment:
Not sure you have much to do about it for now but I expect this log to potentially be verbose.